automation/run-tests.sh

```diff
@@ -179,6 +179,7 @@ function run_tests {
           pytest \
             $PYTEST_OPTIONS \
             tests/integration/static_ip_address_test.py \
+            tests/integration/test_reapply_preserve_ip_config \
```

Comment:
This is incorrect I think you meant -k option here. Please, check https://github.com/nmstate/nmstate/runs/4222094965?check_suite_focus=true#step:6:1600

Reply:
I should use `preserve_ip_config_test.py` instead of test name. Fixed.